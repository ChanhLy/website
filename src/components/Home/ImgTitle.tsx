export interface ImgProps {
  className?: string;
}

export const ImgTitle: React.FC<ImgProps> = (props) => {
  return (
    <div {...props}>
      <div>
        <div className="my-10">
          <div className="my-10 flex justify-between">
            <div className="w-6/12">
              <h1 className="font-bold  text-5xl my-5">Your title here</h1>
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem
                ipsum dolor sit amet, consectetur adipiscing elit.
              </p>
            </div>
            <div className="w-6/12 bg-gray-600 mx-6">
              <div>Your Img</div>
            </div>
          </div>
        </div>
        <div className="my-10">
          <div className="my-60 flex justify-between">
            <div className="w-6/12 bg-gray-600 ">
              <div>Your Img</div>
            </div>
            <div className="w-6/12 mx-6">
              <h1 className="font-bold  text-5xl my-5">Your title here</h1>
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem
                ipsum dolor sit amet, consectetur adipiscing elit.
              </p>
            </div>
          </div>
        </div>
        <div className="my-60">
          <div className="my-10 flex justify-between">
            <div className="w-6/12">
              <h1 className="font-bold  text-5xl my-5">Your title here</h1>
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem
                ipsum dolor sit amet, consectetur adipiscing elit.
              </p>
            </div>
            <div className="w-6/12 bg-gray-600 mx-6">
              <div>Your Img</div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};
