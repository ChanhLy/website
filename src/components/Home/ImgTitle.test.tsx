import React from 'react';

import renderer from 'react-test-renderer';

import { ImgTitle } from './ImgTitle';

it('renders correctly', () => {
  const tree = renderer.create(<ImgTitle></ImgTitle>).toJSON();
  expect(tree).toMatchSnapshot();
});
