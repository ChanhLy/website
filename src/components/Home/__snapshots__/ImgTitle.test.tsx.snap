// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div
      className="my-10"
    >
      <div
        className="my-10 flex justify-between"
      >
        <div
          className="w-6/12"
        >
          <h1
            className="font-bold  text-5xl my-5"
          >
            Your title here
          </h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
        </div>
        <div
          className="w-6/12 bg-gray-600 mx-6"
        >
          <div>
            Your Img
          </div>
        </div>
      </div>
    </div>
    <div
      className="my-10"
    >
      <div
        className="my-60 flex justify-between"
      >
        <div
          className="w-6/12 bg-gray-600 "
        >
          <div>
            Your Img
          </div>
        </div>
        <div
          className="w-6/12 mx-6"
        >
          <h1
            className="font-bold  text-5xl my-5"
          >
            Your title here
          </h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
        </div>
      </div>
    </div>
    <div
      className="my-60"
    >
      <div
        className="my-10 flex justify-between"
      >
        <div
          className="w-6/12"
        >
          <h1
            className="font-bold  text-5xl my-5"
          >
            Your title here
          </h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
        </div>
        <div
          className="w-6/12 bg-gray-600 mx-6"
        >
          <div>
            Your Img
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
